modules/installation-osp-creating-network-resources.adoc

```diff
@@ -21,6 +21,12 @@ include::https://raw.githubusercontent.com/openshift/installer/release-4.6/upi/o
 ----
 include::https://raw.githubusercontent.com/openshift/installer/release-4.6/upi/openstack/inventory.yaml[]
 ----
++
+[TIP]
+====
+If you do not define values for`os_api_fip` and `os_ingress_fip`, you must perform post-installation network configuration. See "Enabling access to the environment" for more information.
```

Comment:
Do you want to add an equivalent for `os_bootstrap_fip`? That would sound like "If you do not define a value for `os_bootstrap_fip`, the installer will not be able to download debug information for failed installations."